### ISSUE TITLE
startpage.com

### ISSUE THREAD

**Issue by Coccocoahelper** [easylist/easylist#18385]
#### List the website(s) you're having issues with:

``startpage.com``

#### What happens?

Ads in Startpage appear

#### Screenshot(s) 

None

#### Filter lists you're using:

uBO filters, EasyList, EasyPrivacy, Online Malicious URL Blocklist, Phishing URL Blocklist, Peter Lowe’s Ad and tracking server list, EasyList – Annoyances, AdGuard Spanish/Portuguese, 🚪 Browse websites without logging in, and Bypass Paywalls Clean filter.

#### Your settings:

- OS/version: Windows 10
- Browser/version: Ungoogled Chromium 120.0.6099.199
- Ad block extension/version: uBlock Origin 1.55.1b11

#### Other details:

Uses Google iFrames, can be fixed by adding ``www.startpage.com###adBlock`` to your filters.

**Comment by partingscientist** [comment]
@Coccocoahelper Can you still reproduce the issue?

**Comment by Coccocoahelper** [comment]
> [@Coccocoahelper](https://github.com/Coccocoahelper) Can you still reproduce the issue?

Nope. 2 years have passed, so it does make sense that this no longer occurs.
